crates/pallets/starknet/src/genesis_loader.rs

```diff
@@ -56,7 +56,7 @@ impl<T: crate::Config> From<GenesisLoader> for GenesisConfig<T> {
                 match class {
                     ContractClass::Path { path, version } => {
                         let contract_path = match loader.madara_path.clone() {
-                            Some(madara_path) => madara_path + "/" + &path,
+                            Some(madara_path) => madara_path + "/configs/" + &path,
                             None => {
                                 let project_path = utils::get_project_path().expect("Project path not found");
```

Comment:
A better expect message could be: "A project path should be present for the node to start"

https://doc.rust-lang.org/std/result/enum.Result.html#method.expect

Reply:
That message may be misleading. What about "Project path not found while trying to load the genesis contracts"

Reply:
"A Project path shoud be present in order to load the genesis contracts"